README.md

```diff
@@ -461,6 +501,9 @@ if (result.error) throw result.error;
 - `toplevel` (default `false`) - set to `true` if you wish to enable top level
   variable and function name mangling and to drop unused variables and functions.
 
+- `nameCache` (default `null`) - pass an object if you wish to cache mangled names
+  across multiple invocations of `minify()`.
```

Comment:
- `nameCache` (default `null`) - pass an empty object `{}` or a previously
  used `nameCache` object if you wish to cache mangled names across multiple
  invocations of `minify()`. Note: this is a read/write property. `minify()`
  will read the name cache state of this object and update this object
  such that it may be reused or externally persisted by the user.